avm/res/cognitive-services/account/main.bicep

```
@@ -339,6 +342,21 @@ resource cognitiveService 'Microsoft.CognitiveServices/accounts@2023-05-01' = {
   }
 }
 
+@batchSize(1)
+resource cognitiveService_deployments 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = [
+  for (deployment, index) in (deployments ?? []): {
+    parent: cognitiveService
+    name: deployment.?name ?? '${name}-deployments'
+    properties: {
+      model: deployment.model
+      raiPolicyName: deployment.?raiPolicyName ?? null
```

```suggestion
      raiPolicyName: deployment.?raiPolicyName
```
Redundant. The `??` selects the first non-null entry. If the former is null, the extra null doesn't do much ;)

If updated please also re-run the `Set-AVMModule` script to re-generate the templates.

---

Updated!